_rules/invalid-form-field-value-36b590.md

```diff
@@ -63,7 +63,7 @@ in [text][] that is [visible][].
 
 ## Expectation 3
 
-For each test target with a [form field error indicator][], at least one of the [form field error indicators][form field error indicator] describes:
+Each test target either has no [form field error indicators][form field error indicator], or at least one of the [form field error indicators][form field error indicator] describes:
```

Comment:
Expectation 2 and 3 are identical ... either Expectation 3 is redundant or should describe something else.